LocalMultiplayerAgent/Config/MultiplayerSettingsValidator.cs

```diff
@@ -35,9 +35,15 @@ public bool IsValid()
                 throw new Exception("OutputFolder or TitleId not found. Call SetDefaultsIfNotSpecified() before this method");
             }
 
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
             {
-                Console.WriteLine("Running LocalMultiplayerAgent on Linux is not supported yet.");
+                Console.WriteLine("Running LocalMultiplayerAgent is not yet supported on MacOS.");
+                return false;
+            }
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) && _settings.RunContainer)
+            {
+                Console.WriteLine("Running LocalMultiplayerAgent as container mode is not yet supported on Linux. Please set RunContainer to false in MultiplayerSettings.json");
```

Comment:
(not for this PR but in general...) this is kind of cumbersome, supporting running Linux Containers under Windows and not under Linux. We should fix that.